public_html/index.html

```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://games.mozilla.org/">
+    <meta property="og:image" content="https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg">
+    <meta property="og:description" content="Mozilla - The Future of Web Games">
+    <meta property="og:title" content="games.mozilla.org">
```

Comment:
you can actually remove this tag. `og:title` I guess is mainly used for individual articles/posts. (news to me!)

Reply:
I'm just going to manually edit this so we can merge this - save you some hassle 😄